### DISCUSSION TITLE
[@types/oojs-ui] Can't pass NumberInputWidget to FieldLayout

### DISCUSSION THREAD

**Discussion by jwbth** [DefinitelyTyped/DefinitelyTyped#71513] [Issues with a @types package]
Compare
```js
new OO.ui.FieldLayout(new OO.ui.NumberInputWidget());

new OO.ui.FieldLayout(new OO.ui.TextInputWidget());
```

![image](https://github.com/user-attachments/assets/e383440b-d696-457f-8c79-e90bfeda81bb)

The error goes away once I remove `Omit` from
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/683ec4a2b420ff6bd3873a7338416ad3ec0b6595/types/oojs-ui/NumberInputWidget.d.ts#L56-L57

[a comment by an automated account is not shown]

**Comment by diskdance** [comment]
See https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/oojs-ui#caveats, this is an OOUI problem.

**Comment by jwbth** [comment]
Ah sorry, didn't notice this specific issue is mentioned there.
